packaging/googet/google-compute-engine-windows.goospec

```diff
@@ -36,7 +37,8 @@
   "sources": [{
       "include": [
         "GCEWindowsAgent.exe",
-	"GCEAuthorizedKeysCommand.exe",
+        "GCEWindowsAgentManager.exe",
+	      "GCEAuthorizedKeysCommand.exe",
```

Comment:
<tab> vs <space> ?

Reply:
Done, its funny actually, IDE and Github is parsing differently, its looks fine only on either of those and not both,
On IDE it looks like this now :)
![Screenshot 2024-10-17 at 4 15 32 PM](https://github.com/user-attachments/assets/a00ce011-c7d7-43ec-aa5a-b3edb22fde5d)